ceph.spec.in

```diff
@@ -470,6 +470,7 @@ BuildArch:      noarch
 Group:          System/Filesystems
 %endif
 Requires:       ceph-mgr = %{_epoch_prefix}%{version}-%{release}
+Requires:       ceph-grafana-dashboards = %{_epoch_prefix}%{version}-%{release}
```

Comment:
Does this have to be added to the Debian packaging, too?

Reply:
Unfortunately, we don't have the dashboards packaged for debian, yet.